multinet/api/tests/test_workspace.py

```diff
@@ -18,20 +25,32 @@ def test_workspace_arango_sync(workspace: Workspace):
 
 @pytest.mark.django_db
 def test_workspace_rest_list(
-    workspace_factory: WorkspaceFactory,
+    public_workspace_factory: PublicWorkspaceFactory,
+    private_workspace_factory: PrivateWorkspaceFactory,
+    user: User,
     authenticated_api_client: APIClient,
 ):
+    """Test list endpoint for workspaces."""
     fake = Faker()
-    workspace_names: List[str] = [workspace_factory(name=fake.pystr()).name for _ in range(3)]
+    accessible_workspace_names: List[str] = [
+        public_workspace_factory(name=fake.pystr()).name for _ in range(3)
+    ]
+    private_workspaces: List[Workspace] = [
+        private_workspace_factory(name=fake.pystr()) for _ in range(3)
+    ]
+
+    private_workspaces[0].set_readers([user])
+    private_workspaces[1].set_owner(user)
```

Comment:
Why specifically set the owner perm here?

Reply:
It was somewhat arbitrary. I'm effectively testing that a workspace shows up in the list view if a user has _exactly_ reader permission (the minimum), and _greater than_ reader permission. Its just an extra thing to test. I could be talked into removing it since its probably redundant. A solution to #56 might solve this issue as well.

Reply:
I think that's fair. I'll go ahead and make that change.